.circleci/config.yml

```diff
@@ -147,7 +147,8 @@ jobs:
             - run:
                 name: Run visual tests
                 command: npm run test.visual
-
+            - store_artifacts:
+                path: storybook-static
```

Comment:
To help figure out what's potentially wrong with visual tests, store the storybook generation